vocab/credentials.jsonld

```diff
@@ -0,0 +1,251 @@
+{
+  "@context": {
+    "id": "@id",
+    "type": "@type",
+    "cred": "https://w3id.org/credentials#",
+    "dc": "http://purl.org/dc/terms/",
+    "schema": "http://schema.org/",
+    "sec": "https://w3id.org/security#",
+    "xsd": "http://www.w3.org/2001/XMLSchema#",
+    "CryptographicKey": "sec:Key",
+    "GraphSignature2012": "sec:GraphSignature2012",
+    "Identity": "https://w3id.org/identity#Identity",
+    "LinkedDataSignature2015": "sec:LinkedDataSignature2015",
+    "created": {
+      "@id": "dc:created",
+      "@type": "xsd:dateTime"
+    },
+    "creator": {
+      "@id": "dc:creator",
+      "@type": "@id"
+    },
+    "domain": "sec:domain",
+    "expires": {
+      "@id": "sec:expiration",
+      "@type": "xsd:dateTime"
+    },
+    "nonce": "sec:nonce",
+    "normalizationAlgorithm": "sec:normalizationAlgorithm",
+    "owner": {
+      "@id": "sec:owner",
+      "@type": "@id"
+    },
+    "privateKey": {
+      "@id": "sec:privateKey",
+      "@type": "@id"
+    },
+    "privateKeyPem": "sec:privateKeyPem",
+    "publicKey": {
+      "@id": "sec:publicKey",
+      "@type": "@id"
+    },
+    "publicKeyPem": "sec:publicKeyPem",
+    "publicKeyService": {
+      "@id": "sec:publicKeyService",
+      "@type": "@id"
+    },
+    "revoked": {
+      "@id": "sec:revoked",
+      "@type": "xsd:dateTime"
+    },
+    "signature": "sec:signature",
+    "signatureAlgorithm": "sec:signatureAlgorithm",
+    "signatureValue": "sec:signatureValue",
+    "Credential": "cred:Credential",
+    "claim": {
```

Comment:
This needs to be updated in the test suite and here - "@container": "@graph", I think. @dlongley, I forget the details, but it looks like it's wrong in both the test suite and here.

Reply:
I'll add it, but we haven't defined `@container: @graph` in the spec, yet.